near-sdk/src/environment/mock/mocked_blockchain.rs

```diff
@@ -102,7 +103,7 @@ impl MockedBlockchain {
     }
 
     pub fn gas(&mut self, gas_amount: u32) {
-        self.logic.borrow_mut().gas(gas_amount).unwrap()
+        self.logic.borrow_mut().gas(gas_amount as u64).unwrap()
```

Comment:
I think `.into()` should work, and it will be safer than type casting as if in the future we change the type of `gas_amount` we might forget about this type cast.

```suggestion
        self.logic.borrow_mut().gas(gas_amount.into()).unwrap()
```